src/test/java/com/hazelcast/aws/DescribeInstancesTest.java

```diff
@@ -1,51 +0,0 @@
-/*
```

Comment:
I am just checking from github web UI and it is not easy but if you squash your commits then that can be easy.